### ISSUE TITLE
Wunsch: Zusammenfügen JL-DB mit JSM-DB

### ISSUE THREAD

**Issue by svdoldie** [diddipoeler/sportsmanagement#878]
Ich habe folgendes Problem. Ich habe meine JL-Datenbank so aufgeräumt, dass sie jetzt importbereit ist.
Allerdings habe ich keine Möglichkeit sie mit den mittlerweile vorhanden JSM-Projekten bzw. JSM-DB zusammenzufügen. Meine Vorstellung geht dahin, das der JSM-Export eines Projekts inkl. aller Daten ermöglicht wird, um diese dann nach einen JL-DB-Imports wieder zu importieren. Aber vielleicht gibt es ja auch eine andere Möglichkeit beide DBs zusammenzufügen.

**Comment by diddipoeler** [comment]
Wie ?
JSM Daten sollen in eine Joomleague Datenbank importiert werden ?

**Comment by svdoldie** [comment]
Nein. Es sollen die JL-DB in die JSM-DB importiert werden, aber die vorhanden JSM-Daten erhalten bleiben.
Oder eben erst die JSM-Projekte exportiert werden, und nach Leerung der JSM-Datenbank und anschließendem JL-DB-Import wieder importiert werden.

**Comment by diddipoeler** [comment]
Ah verstanden. Aber enormer Programmieraufwand.
Ich würde folgende Reihenfolge vorschlagen:
Es sollen die JL-DB in die JSM-DB importiert werden, aber die vorhanden JSM-Daten erhalten bleiben.

**Comment by svdoldie** [comment]
Das wäre natürlich top, wenn das möglich wäre !!! :-)

**Comment by svdoldie** [comment]
Es gibt übrigens ein oder  zwei Probleme mit dem JL-DB-Import.
Soll ich die hier eintragen, oder besser als eigene issues?

**Comment by diddipoeler** [comment]
> 
> 
> Es gibt übrigens ein oder zwei Probleme mit dem JL-DB-Import.
> Soll ich die hier eintragen, oder besser als eigene issues?

Eigene Issues !

**Comment by svdoldie** [comment]
Geht los ;-)

**Comment by svdoldie** [comment]
Moin Diddi,
gibt's ne Chance, dass du das bis zur neuen Saison gelöst bekommst?
Gruß svdoldie
